collect_app/src/main/java/org/odk/collect/android/formlists/savedformlist/DeleteSavedFormFragment.kt

```diff
@@ -36,7 +36,7 @@ class DeleteSavedFormFragment(
     private val multiSelectViewModel: MultiSelectViewModel<Instance> by viewModels {
         MultiSelectViewModel.Factory(
             savedFormListViewModel.formsToDisplay.map {
-                it.filter { instance -> instance.canDeleteBeforeSend() }
+                it.filter { instance -> instance.canDelete() }
```

Comment:
The filter here previously didn't consider instance state at all so I don't think there's any way it could have been excluding drafts.